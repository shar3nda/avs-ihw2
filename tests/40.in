C6
Wx,|/<N	i<=C:GKLAr+O+q4w<LDx$k76	o3<QyKrFd7M|C Jv#TiY~]r1Ae|
Z}+j\LzK>%