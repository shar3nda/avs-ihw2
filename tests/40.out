Lowercase: 898, uppercase: 854